src/elements/common/interactionTargets.js

```diff
@@ -28,8 +28,8 @@ export const DETAILS_TARGETS = {
 };
 
 export const ACTIVITY_TARGETS = {
-    INLINE_EDIT: 'activityfeed-inlineedit',
-    INLINE_DELETE: 'activityfeed-inlinedelete',
+    INLINE_DELETE_CONFIRM: 'activityfeed-delete-confirm',
```

Comment:
Will this cause any issues with existing event records or were they broken previously due to the missing component attribute?

Reply:
The old variables were not referenced anywhere and the new ones were not reporting in